spot-ml/src/main/scala/org/apache/spot/netflow/model/FlowSuspiciousConnectsModel.scala

```diff
@@ -184,7 +188,7 @@ object FlowSuspiciousConnectsModel {
 
     // simplify DNS log entries into "words"
 
-    val flowWordCreator = new FlowWordCreator(ibytCuts, ipktCuts, timeCuts)
+    val flowWordCreator = new FlowWordCreator(timeCuts, ibytCuts, ipktCuts)
```

Comment:
Was this an existing error? Have we been passing the wrong parameters to FlowWordCreator all the time? Or did FlowWordCreator constructor changed?